_includes/references.md

```diff
@@ -28,24 +28,31 @@
 [bolts repo]: https://github.com/lightningnetwork/lightning-rfc/
 [c-lightning]: https://github.com/ElementsProject/lightning
 [c-lightning repo]: https://github.com/ElementsProject/lightning
+[eclair repo]: https://github.com/ACINQ/eclair
+[hwi repo]: https://github.com/bitcoin-core/HWI
+[libminisketch]: https://github.com/sipa/minisketch
+[libsecp256k1]: https://github.com/bitcoin-core/secp256k1
+[libsecp256k1 repo]: https://github.com/bitcoin-core/secp256k1
+[lnd repo]: https://github.com/lightningnetwork/lnd/
+[rust-lightning repo]: https://github.com/rust-bitcoin/rust-lightning
+
+{% comment %}<!-- deprecated links; don't use these any more -->{% endcomment %}
+{% assign pagedate_epoch = page.date | date: '%s' %}
+{% assign deprecated_links_v0_epoch = '2020-09-24' | date: '%s' %}
```

Comment:
What's the plan if we want to deprecate more links in future. Does this date need to be updated?

Reply:
My idea was that, if we want to deprecate links en masse in the future, we'd create a v1 deprecation by copying this template.  E.g. 

    {% assign pagedate_epoch = page.date | date: '%s' %}
    {% assign deprecated_links_v0_epoch = '2020-09-24' | date: '%s' %}
    {% if pagedate_epoch < deprecated_links_v0_epoch %}
    [... links deprecated in this PR ...]
    {% endif %}

    {% assign pagedate_epoch = page.date | date: '%s' %}
    {% assign deprecated_links_v2_epoch = '2022-01-01' | date: '%s' %}
    {% if pagedate_epoch < deprecated_links_v0_epoch %}
    [... links deprecated in future PR ...]
    {% endif %}